examples/echo/jsfiddle/demo.js

```diff
@@ -13,7 +13,13 @@ var log = msg => {
 
 navigator.mediaDevices.getUserMedia({ video: true, audio: true })
   .then(stream => {
-    pc.addStream(stream)
+    if (pc.addTrack) {
```

Comment:
IMO we can drop `addStream` and just do `addTrack`!

Mind changing it everywhere? Also feel free to use arrow functions (if you want) your PR :)

`stream.getTracks().forEach(track => pc.AddTrack(stream, stream)`

Reply:
OK, I will do that. I was not sure if that would break something really old...

Reply:
Yea totally!

I would say don't worry about using modern features in the examples. The only time I really care is when we change the API, refactoring aggressively in the examples or internal API is a fun free for all :)